test/lightning/projects_test.exs

```diff
@@ -153,4 +224,76 @@ defmodule Lightning.ProjectsTest do
       assert generated_yaml == expected_yaml
     end
   end
+
+  describe "The default Oban function Projects.perform/1" do
+    test "finds all project users of a project that has a digest of :daily, :weekly, and :monthly" do
+      user_1 = user_fixture()
+      user_2 = user_fixture()
+      user_3 = user_fixture()
+
+      project_fixture(
+        project_users: [
+          %{user_id: user_1.id, digest: :daily},
+          %{user_id: user_2.id, digest: :daily},
+          %{user_id: user_3.id, digest: :weekly}
+        ]
+      )
+
+      {:ok, %{project_users: daily_project_users}} =
+        Projects.perform(%Oban.Job{args: %{"type" => "daily_project_digest"}})
+
+      {:ok, %{project_users: weekly_project_users}} =
+        Projects.perform(%Oban.Job{args: %{"type" => "weekly_project_digest"}})
+
+      {:ok, %{project_users: monthly_project_users}} =
+        Projects.perform(%Oban.Job{args: %{"type" => "monthly_project_digest"}})
+
+      assert daily_project_users |> length() == 2
+      assert weekly_project_users |> length() == 1
+      assert monthly_project_users |> length() == 0
+    end
+
+    test "finds all project users of different projects that has a digest of :daily, :weekly, and :monthly" do
+      user_1 = user_fixture()
+      user_2 = user_fixture()
+      user_3 = user_fixture()
+
+      project_fixture(
+        project_users: [
+          %{user_id: user_1.id, digest: :daily},
+          %{user_id: user_2.id, digest: :weekly},
+          %{user_id: user_3.id, digest: :monthly}
+        ]
+      )
+
+      project_fixture(
+        project_users: [
+          %{user_id: user_1.id, digest: :monthly},
+          %{user_id: user_2.id, digest: :weekly},
+          %{user_id: user_3.id, digest: :daily}
+        ]
+      )
+
+      project_fixture(
+        project_users: [
+          %{user_id: user_1.id, digest: :monthly},
+          %{user_id: user_2.id, digest: :daily},
+          %{user_id: user_3.id, digest: :weekly}
+        ]
+      )
+
+      {:ok, %{project_users: daily_project_users}} =
+        Projects.perform(%Oban.Job{args: %{"type" => "daily_project_digest"}})
+
+      {:ok, %{project_users: weekly_project_users}} =
+        Projects.perform(%Oban.Job{args: %{"type" => "weekly_project_digest"}})
+
+      {:ok, %{project_users: monthly_project_users}} =
+        Projects.perform(%Oban.Job{args: %{"type" => "monthly_project_digest"}})
+
+      assert daily_project_users |> length() == 3
+      assert weekly_project_users |> length() == 3
+      assert monthly_project_users |> length() == 3
```

Comment:
Perhaps mix these counts up, this doesn't prove that you're _not_ just getting all the project users for a project. Have at least one of the project users have no digest settings.